kinetic/distribution.yaml

```diff
@@ -5028,6 +5028,23 @@ repositories:
       url: https://github.com/astuff/ibeo_lux.git
       version: release
     status: developed
+  ifm3d-release:
```

Comment:
This is an odd name for a key.  Usually the key is either the name of the repository (ifm3d), or the name of the package (ifm3d_core).

Reply:
hmm yes I agree, sorry for the oversight. This PR was auto-generated by `bloom-release`. This is my first release so pardon my learning curve -- shall I simply modify the PR manually or is there some metadata in my [tracks file](https://github.com/ifm/ifm3d-release/blob/master/tracks.yaml) that's missing which ought to specify this key name?

Reply:
Or is it the `repository` parameter in the `bloom-release` command that I should have set to ifm3d_core?

```
usage: bloom-release [-h] [--list-tracks] [--track TRACK] [--non-interactive]
                     --ros-distro ROS_DISTRO [--new-track] [--pretend]
                     [--no-pull-request] [--no-web] [--pull-request-only]
                     [--override-release-repository-url OVERRIDE_RELEASE_REPOSITORY_URL]
                     [--override-release-repository-push-url OVERRIDE_RELEASE_REPOSITORY_PUSH_URL]
                     [-d] [--version] [--unsafe]
                     repository
```